lib/rbi/parser.rb

```diff
@@ -648,10 +648,11 @@ def visit(node)
   class SigBuilder < ASTVisitor
     extend T::Sig
 
+    sig { void }
```

Comment:
@Morriar Is there a way to provide feedback to code-db for a mistake case like this?

Reply:
I'm already looking at fixing this 👍 

But the general approach if to comment and close. Leave this one open for now though, I'll make CodeDb force push it with the fix.